src/lib.rs

```diff
@@ -2,6 +2,7 @@
 #![allow(non_upper_case_globals)]
 
 #![cfg_attr(not(feature = "use_std"), no_std)]
+#![cfg_attr(feature = "unstable", feature(repr_packed))]
```

Comment:
`packed` is opt-in, users compiling with nightly need to explicitly opt-into it, this allows the library to keep working as is on stable Rust.